common/components/controllers/LogInController.jsx

```diff
@@ -23,7 +23,7 @@ class LogInController extends React.Component<Props, State> {
     this.state = {
       username: '',
       password: '',
-      prevPage: window.location.href.split('&prev=')[1] || this.props.prevPage || document.referrer.split('section=')[1] || ''
+      prevPage: window.location.href.split('&prev=')[1] || this.props.prevPage || ''
```

Comment:
One nice thing about document.referrer we might want to keep in mind for the future is that it required passing no props or additional information in the query string, it 'just worked' (or didn't) automatically. 

Checking the PROTOCOL_DOMAIN env key to ensure only internal referrers are parsed here may be valuable if we choose to re-enable this case down the line - though it's not required at this time, since we changed how the Log In links are constructed in the nav to begin with.